src/servers/sapphire_zone/Action/ActionCast.cpp

```diff
@@ -31,7 +31,7 @@ Core::Action::ActionCast::ActionCast( Entity::ActorPtr pActor, Entity::ActorPtr
    m_startTime = 0;
    m_id = actionId;
    m_handleActionType = HandleActionType::Spell;
-   m_castTime = g_exdData.getActionInfo( actionId )->cast_time; // TODO: Add security checks.
+   m_castTime = g_exdDataGen.getAction( actionId )->cast100ms; // TODO: Add security checks.
```

Comment:
whenever this is used, it now needs to be *100 iirc as that is how it was originally stored as discussed earlier